Cython/Compiler/ModuleNode.py

```diff
@@ -3076,7 +3076,8 @@ def generate_module_init_func(self, imported_modules, env, code):
         code.put_setup_refcount_context(header3)
 
         env.use_utility_code(UtilityCode.load("CheckBinaryVersion", "ModuleSetupCode.c"))
-        code.put_error_if_neg(self.pos, "__Pyx_check_binary_version()")
+        code.put_error_if_neg(
+            self.pos, "__Pyx_check_binary_version(PY_VERSION_HEX, __Pyx_get_runtime_version(), CYTHON_COMPILING_IN_LIMITED_API)")
```

Comment:
`__PYX_LIMITED_VERSION_HEX` instead of `Py_VERSION_HEX` I think. They're equivalent in regular mode, but in limited API mode it's the version of limited API you're using